formcentral/go.mod

```diff
@@ -4,9 +4,12 @@ go 1.15
 
 require (
 	github.com/caarlos0/env/v6 v6.4.0
+	github.com/fsnotify/fsnotify v1.5.1 // indirect
```

Comment:
I wonder if there is an equivalent in `go` to `devDependencies`, like in npm world.

Reply:
There isn't, but i would leave it out of here, it can just be in the dev container if we go that route.

Reply:
_changes reverted here https://github.com/vlab-research/fly/pull/23/commits/442126a4457483617cc60b605cfb27425cb9020e_